mlflow/utils/databricks_utils.py

```diff
@@ -50,6 +79,7 @@ def _get_context_tag(context_tag_key):
         return None
 
 
+@_use_env_var_if_exists(_env_var_prefix + "ACL_PATH_OF_ACL_ROOT")
```

Comment:
Can we prefix these environment variables with `DATABRICKS`?

Reply:
We probably don't need `ACL_PATH_OF_ACL_ROOT`, since this is used for `is_in_databricks_notebook` / `get_notebook_id`. We can rely on `DATABRICKS_NOTEBOOK_ID` for those.

Reply:
@dbczumar Thanks for the comment! `_env_var_prefix` adds `DATABRICKS_` or am I missing something?

Reply:
Doh. Sorry - missed that.

Reply:
> We probably don't need ACL_PATH_OF_ACL_ROOT, since this is used for is_in_databricks_notebook / get_notebook_id. We can rely on DATABRICKS_NOTEBOOK_ID for those.

Makes sense!